modules/accounts/accounts.js

```diff
@@ -92,6 +99,17 @@ class Accounts{
 
     setProviderAccounts(args){    
         config.provider = args[0];
+
+        if(os.platform().includes(WIN_OS_PLATFORM) && config.provider.includes('localhost')) {
```

Comment:
Move this 'os.platform().includes' to a helper function 'isWIndows' and resuse that everywhere.